HLTrigger/special/src/HLTTrackerHaloFilter.cc

```diff
@@ -58,7 +57,7 @@ bool HLTTrackerHaloFilter::hltFilter(edm::Event& event, const edm::EventSetup& i
 /*
   //Retrieve tracker topology from geometry
   edm::ESHandle<TrackerTopology> tTopoHandle;
-  iSetup.get<IdealGeometryRecord>().get(tTopoHandle);
+  iSetup.get<TrackerTopologyRcd>().get(tTopoHandle);
   const TrackerTopology* const tTopo = tTopoHandle.product();
```

Comment:
No, it looks like line 61 is replaced(!) by new line 60.
So there is an active chnage, and removal of line 14 is only because it is no longer needed
as line 61 is removed...
??